Shadows.md

```diff
@@ -0,0 +1,19 @@
+
+### Differences in Shadows across platforms
+- Figma uses the same shadow properties as web: i.e. the blur property
+- react-native-iOS uses shadowRadius. In order to get iOS shadows to look the same as Figma, shadowRadius should be blur/2. Native iOS should also use blur/2, as well as react-native-macOS
```

Comment:
what about native macos?

Reply:
Did not test on native macOS or native iOS, but I can do that

Reply:
I would de-emphasize "React Native" per say, and maybe just say "Apple (iOS and macOS), Android, and Web". In a  separate sentence, we can say "This applies to these platforms on React Native as well (React Native, React Native macOS, and React Native Web), with a link to your snack

Reply:
Or maybe even have a section named "What about React Native? :P

Reply:
Add macOS, links to the snack I used, and info about react native throughout. If you think a section dedicated to RN is still needed lmk